### ISSUE TITLE
Do basic a11y audit

### ISSUE THREAD

**Issue by pirog** [kalabox/tandem#58]

**Comment by pirog** [comment]
added a test for this but set to pass on fail, it looks like addressing the issues with `a11y` and `sass` will be a long term project that we might want to tackle a bit at a time.
